website/app/controllers/index.js

```diff
@@ -7,18 +7,18 @@ import Controller from '@ember/controller';
 
 export default class IndexController extends Controller {
   cards = [
-    {
-      title: 'About',
-      description:
-        'Our aims, principles, and strategy and how to get started with Helios.',
-      route: 'about',
-    },
```

Comment:
We don't want to include the About card anymore?

Reply:
[Example of this here.](https://www.figma.com/design/ZlIlup8HxCOlQLXRwv56In/HDS-Content-Guidelines?node-id=3-11597&t=uKAjr2LKxywzroAz-1)